Sources/ProjectDescription/Dependencies/SwiftPackageManagerDependencies.swift

```diff
@@ -1,21 +1,24 @@
 import Foundation
 
+public enum PackagesOrManifestPath: Codable, Equatable {
+    case packages([Package])
+    case manifest(Path)
+}
+
 /// A collection of Swift Package Manager dependencies.
 ///
 /// For example, to enabled resource accessors on projects generated from Swift Package Manager:
 ///
 /// ```swift
 /// let packageManager = SwiftPackageManagerDependencies(
-///     packages: [
-///         .local(path: "MySwiftPackage")
-///     ],
+///     "Package.swift",
 ///     projectOptions: ["MySwiftPackage":  .options(disableSynthesizedResourceAccessors: false)]
 /// )
 /// ```
-
 public struct SwiftPackageManagerDependencies: Codable, Equatable {
-    /// List of packages that will be installed using Swift Package Manager.
-    public let packages: [Package]
+    /// The path to the `Package.swift` manifest defining the dependencies, or the list of packages that will be installed using
+    /// Swift Package Manager.
+    public let packagesOrManifestPath: PackagesOrManifestPath
```

Comment:
Is there a need to make this configurable? Shouldn't users either:
- Use `Dependencies.swift` (deprecated) -> and only use the `packages: [Package]` init
- Use `Package.swift` that is in `Tuist/Dependencies/Package.swift`

Reply:
The whole `Tuist/Dependencies/` folder is supposed to not be committed as of now, so for now I have preferred to avoid that, but no strong opinion